pkg/operator/ceph/object/admin.go

```diff
@@ -205,6 +205,10 @@ func extractJSON(output string) (string, error) {
 	return string(objMatch), nil
 }
 
+// IS_UNIT_TEST is really poor practice but allows us to avoid rewriting an extremely complex unit test
+// this is as hidden as possible so it's harder to follow my bad example
+var IS_UNIT_TEST = false
```

Comment:
Instead of a bool, what about three modes? While it is a bit more complex, it seems worth it. 
- `production`: Local call, then if it fails fall back to the remote call
- `remoteOnly`: Only make the remote call. Some integration test suites could enable this so that we get test coverage on the remote execution
- `localOnly`: Would be used by unit tests (I believe as you are using it here)

Reply:
I can make it an env var. I guess it's named like it is one. Just a global var.

Reply:
I took this idea but adjusted it to fit a little better. Instead it is `remoteOnly` and `doNotFallBackToRemote`. It is assumed `production` is the empty string instead. I think this helps keep the logic simpler.

If users start to see an issue with this code in production, we can direct them to use `doNotFallBackToRemote`. With that, the behavior should be the same as before, with the sidecar and proxy only used when multus is enabled.

Unit tests can use `doNotFallBackToRemote` as well. Integration tests can use `remoteOnly` if desired, to stress the proxy code.